package.json

```diff
@@ -52,7 +52,8 @@
     "debug": "~4.3.2",
     "engine.io": "~6.5.2",
     "socket.io-adapter": "~2.5.2",
-    "socket.io-parser": "~4.2.4"
+    "socket.io-parser": "~4.2.4",
+    "type-fest": "^3.13.1"
```

Comment:
We try to keep our prod dependencies to the bare minimum, so I think we'll indeed need to copy the types from `type-fest`.